### DISCUSSION TITLE
External connections

### DISCUSSION THREAD

**Discussion by credondocr** [air-verse/air#225] [Q&A]
I'm trying to allow external connections to air because when I reach through the IP address, the server does not respond, there's a way to allow external connections?

**Comment by xiantang** [comment]
Could u provide more detailed infomation?

**Reply by credondocr** [reply]
Sure, I'm running my code with air on amazon (just to test it ) when I try to hit the endpoint from my computer(in another network) I cannot reach them because the port is not open to external connections with air.
If I run the code using the default command without hot reload (go run main.go)  I can reach them from anywhere.
I want this feature just to test my code in a testing server just fetching the new code and hot reload them with air.

Thanks
